src/View/Cell.php

```diff
@@ -259,15 +260,17 @@ protected function _cacheConfig($action)
      * This is because PHP will not allow a __toString() method to throw an exception.
      *
      * @return string Rendered cell
+     * @throws \Error Include error details for PHP 7 fatal errors.
      */
     public function __toString()
     {
         try {
             return $this->render();
         } catch (Exception $e) {
             trigger_error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()), E_USER_WARNING);
-
             return '';
+        } catch (Error $e) {
+            throw new Error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()));
```

Comment:
@markstory So my current believe is that PHP 7 fatal errors should not be caught. If you re-throw the error than Cake's error handler will report it correctly. If you go with `trigger_error` we rish introducing the bug where transactions are not rolled back for fatal errors.

No, I strongly advise that the fatal error is thrown and things continue to work as they have in previous versions of PHP. If we use trigger_error here we introduce other bugs. Trust me.